dotnet/src/Microsoft.AutoGen/Agents/Services/Orleans/iSubscriptionsGrain.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// iSubscriptionsGrain.cs
+
+namespace Microsoft.AutoGen.Agents;
+public interface ISubscriptionsGrain : IGrainWithIntegerKey
```

Comment:
Can we make it internal?

Reply:
Also, can you capitalize the first character of this file name

`iSubscriptionsGrain.cs -> ISubscriptionsGrain.cs`

Reply:
this one (all the grains) are not necessarily internal?
renamed